### ISSUE TITLE
Implementing the subscriber and publisher (for Worker module - TurtleBot)

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#56]
## User story
1. As a SD
2. I want ro Implementing the publisher and the subscriber
3. So that I can get relevant topics from the mqtt node on the robot, the localisation topic and the line measurement topic. Later I can publish the current location, speed and direction.

## Acceptance criteria
* Get relevant topics from the mqtt node on the robot, the localisation topic and the line measurement topic
* Publish the current location, speed and direction

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by MeikeBloecher** [comment]
Subscribers: LocalizationControllerResultMessage0502, LineMeasurementMessage0403, Order, EnergyLevel
Publisher: AGV Position, MotorPower
